veracruz-server/src/server.rs

```diff
@@ -132,22 +132,22 @@ pub fn server(policy_json: &str) -> Result<Server, VeracruzServerError> {
         App::new()
             // pass in the shutdown channel and enclave handler VERACRUZ_SERVER to the server
             .wrap(middleware::Logger::default())
-            .data(shutdown_channel_tx.clone())
-            .data(VERACRUZ_SERVER.clone())
+            .app_data(web::Data::new(shutdown_channel_tx.clone()))
+            .app_data(web::Data::new(VERACRUZ_SERVER.clone()))
             .service(veracruz_server_request)
             .service(runtime_manager_request)
     })
     .bind(&policy.veracruz_server_url())?
     .run();
 
     // clone the Server handle and pass the the thread for shuting down the server
```

Comment:
Update comment:
Server isn't cloned anymore
Fix mistakes: the the, shuting

Reply:
Fixed